### PR TITLE
efi image builder doc

### PR DESCRIPTION
**Pull request by ddjjia**
*Issue #, if available:*
https://github.com/aws/eks-anywhere-internal/issues/1604

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/osmgmt/artifacts.md [hunk 1]
```diff
@@ -433,8 +433,9 @@ These steps use `image-builder` to create an Ubuntu-based or RHEL-based image fo
 
       * `--os`: `ubuntu`
       * `--hypervisor`: For vSphere use `vsphere`
-      * `--release-channel`: Supported EKS Distro releases include 1-21, 1-22, 1-23, 1-24 and 1-25.
+      * `--release-channel`: Supported EKS Distro releases include 1-21, 1-22, 1-23, 1-24 and 1-25
       * `--vsphere-config`: vSphere configuration file (`vsphere-connection.json` in this example)
+      * `--firmware efi`: (optional) To enable EFI in Ubuntu. This is only supported for vSphere Ubuntu image
```
Comment (chrisnegus) [inline docs/content/en/docs/osmgmt/artifacts.md:438]:
```suggestion
      * `--firmware efi`: (optional) To enable EFI in Ubuntu. This is only supported for vSphere Ubuntu images
```

**Review by chrisnegus — COMMENTED** [review]
Small suggestion.

File: docs/content/en/docs/osmgmt/artifacts.md [hunk 2]
```diff
@@ -443,7 +444,7 @@ These steps use `image-builder` to create an Ubuntu-based or RHEL-based image fo
 
       * `--os`: `redhat`
       * `--hypervisor`: For vSphere use `vsphere`
-      * `--release-channel`: Supported EKS Distro releases include 1-21, 1-22, 1-23, 1-24 and 1-25.
+      * `--release-channel`: Supported EKS Distro releases include 1-21, 1-22, 1-23, 1-24 and 1-25
```
Comment (jaxesn) [inline docs/content/en/docs/osmgmt/artifacts.md:447]: Can we go ahead and update these list to include 1.26 and 1.27?

File: docs/content/en/docs/osmgmt/artifacts.md [hunk 3]
```diff
@@ -433,21 +433,22 @@ These steps use `image-builder` to create an Ubuntu-based or RHEL-based image fo
 
       * `--os`: `ubuntu`
       * `--hypervisor`: For vSphere use `vsphere`
-      * `--release-channel`: Supported EKS Distro releases include 1-21, 1-22, 1-23, 1-24 and 1-25.
+      * `--release-channel`: Supported EKS Distro releases include 1-23, 1-24 1-25 1-26 and 1-27
       * `--vsphere-config`: vSphere configuration file (`vsphere-connection.json` in this example)
+      * `--firmware efi`: (optional) To enable EFI in Ubuntu. This is only supported for vSphere Ubuntu images
```
Comment (jiayiwang7) [inline docs/content/en/docs/osmgmt/artifacts.md:438]:
```suggestion
      * `--firmware-efi`: (optional) To enable EFI in Ubuntu. This is only supported for vSphere Ubuntu images
```

?

**Pull request merged**